code/modules/spacepods/spacepod.dm

```diff
@@ -130,7 +130,7 @@
 	light_color = icon_light_color[src.icon_state]
 
 /obj/spacepod/bullet_act(var/obj/item/projectile/P)
-	if(P.damage && !P.nodamage)
+	if(P.damage && !P.nodamage && P.damage_type != STAMINA)
```

Comment:
would be better to check for `if((Proj.damage_type == BRUTE || Proj.damage_type == BURN))`

I don't think clone damage should hurt pods =p